tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/DownloadFileSB.cs

```diff
@@ -0,0 +1,304 @@
+using Microsoft.Build.Framework;
```

Comment:
I recommend adding a comment with a pinned link (press <kbd>y</kbd> on the target GitHub page) to where this came from. Also describe any changes to the body.